test/foundry/strategies/DirectGrantsSimpleStrategy.t.sol

```diff
@@ -14,6 +14,7 @@ import {AlloSetup} from "../shared/AlloSetup.sol";
 import {RegistrySetupFull} from "../shared/RegistrySetup.sol";
 
 import {EventSetup} from "../shared/EventSetup.sol";
+import {Hats} from "hats-protocol/Hats.sol";
```

Comment:
TODO: remove this before merging if we are alright with the reampping update @0xKurt @codenamejason

Reply:
```suggestion
```